service-worker.js

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+const VERSION = 1;
+
+toolbox.cache.name = "Babel-Cache-" + VERSION;
```

Comment:
Rather than cleaning manually the cache, I think we need to limit the cache using `cache.maxEntries` or `cache.maxAgeSeconds`.
The old cached website(s) will be cleaned up given these configurations.